src/content_mapblock.cpp

```diff
@@ -172,6 +172,10 @@ static inline void getNeighborConnectingFace(v3s16 p, INodeDefManager *nodedef,
 		*neighbors |= v;
 }
 
+static int NeighbourToIndex(const v3s16 &pos) {
```

Comment:
Should there be a suggestion to the compiler/linker that this is better inlined?